cms/jinja2/templates/components/streamfield/correction_block.html

```diff
@@ -1,9 +1,12 @@
-<div>
-    {{ value.when|date("DATETIME_FORMAT") }}
-
-    {{ value.text|richtext }}
-
-    <a href="{{ routablepageurl(page, 'previous_version', value.version_id) }}">
-        {{ _("View superseded version") }}
-    </a>
-</div>
+<div class="ons-corrections-notices__item">
+    <div class="ons-corrections-notices__content-meta">
+        <h3 class="ons-u-fs-r--b ons-corrections-notices__content-heading">{{ _("Correction") }}</h3>
+        <time class="ons-corrections-notices__content-date">{{ value.when|date("DATETIME_FORMAT") }}</time>
+    </div>
+    <div class="ons-corrections-notices__content-description">
+        <div class="ons-corrections-notices__content-text ons-u-mb-s">{{ value.text|richtext }}</div>
+        <a class="ons-corrections-notices__content-url" href="{{ routablepageurl(page, 'previous_version', value.version_id) }}">
+            {{ _("View superseded version") }}
+        </a>
+    </div>
+</div>
```

Comment:
Nit-pick: I think we tend to leave one blank line at the end of each file. If you use VSCode, you can configure it to do that for you by setting `files.insertFinalNewline` to true in your settings (Github highlights this with the red circle with a line icon)

Reply:
Yeah good shout thank you - have run formatting now and all sorted 👍

Reply:
And added comments around the include - I think this was my missing piece!

Reply:
I'm afraid I still see it in the first two files, did you forget to push?

Reply:
Hm I must have, it should be updated now.